jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -53,6 +53,12 @@ class SpectralExtraction(PluginTemplateMixin, DatasetSelectMixin,
     extracted_spec_available = Bool(False).tag(sync=True)
     overwrite_warn = Bool(False).tag(sync=True)
 
+    aperture_method_items = List(['exact', 'subpixel', 'center']).tag(sync=True)
+    aperture_method_selected = Unicode('exact').tag(sync=True)
+    cone_aperture_slope = Float().tag(sync=True)
+    cone_aperture_intercept = Float().tag(sync=True)
+    cone_aperture_center = Any().tag(sync=True)
```

Comment:
we definitely don't need slope or intercept and should be able to pull all the information from the reference subset.  Once this is rebased on #2664 (or preferably #2676 if that is accepted), then this should be able to make use of the same traitlets defined there to define the cone itself.